pkg/solana/logpoller/types.go

```diff
@@ -240,7 +253,15 @@ func newIndexedValue(typedVal any) (iVal IndexedValue, err error) {
 	// any length array is fine as long as the element type is byte
 	if t := v.Type(); t.Kind() == reflect.Array {
 		if t.Elem().Kind() == reflect.Uint8 {
-			return v.Bytes(), nil
+			if v.CanAddr() {
```

Comment:
Wow, interesting! Had to read up on this a bit just now to understand why this is necessary, but I think I get it.